utility/ArduinoBLE_UART_Stream.cpp

```diff
@@ -0,0 +1,3 @@
+/*
+ * Implementation is in ArduinoBLE_UART_Stream.h to avoid linker issues.
```

Comment:
Are you sure this is necessary? What kind of linker errors do you see when you do it the "normal" way?

Reply:
The issue is that, if you move the class implementation to the cpp file, the Arduino IDE will *always* compile it, even if the current project doesn't require it and/or the current board is not compatible with it.

For example, if the method implementations are in the cpp file (i.e. the "normal" way), and you try to build StandardFirmata for an Arduino Uno, compilation fails because ArduinoBLE is missing:
```
ArduinoBLE_UART_Stream.cpp:5:0:
/Users/cstawarz/Documents/Arduino/libraries/Firmata/utility/ArduinoBLE_UART_Stream.h:11:10: fatal error: ArduinoBLE.h: No such file or directory
 #include <ArduinoBLE.h>
          ^~~~~~~~~~~~~~
compilation terminated.
```
If you then install ArduinoBLE, even though you obviously don't need it, you get more build failures:
```
/Users/cstawarz/Documents/Arduino/libraries/ArduinoBLE/src/utility/HCIUartTransport.cpp:33:2: error: #error "Unsupported board selected!"
 #error "Unsupported board selected!"
  ^~~~~
/Users/cstawarz/Documents/Arduino/libraries/ArduinoBLE/src/utility/HCIUartTransport.cpp:99:40: error: 'SerialHCI' was not declared in this scope
 HCIUartTransportClass HCIUartTransport(SerialHCI, 912600);
                                        ^~~~~~~~~
```

The "avoid linker issues" bit was just a lazy copy from BLEStream.cpp.  Should I change it to say "avoid an unnecessary dependency on the ArduinoBLE library"?

Reply:
Ok, that makes sense, even though it's a bit ugly. Yes, please change that comment, then it will be fine for me.

Reply:
Done!  I also changed the comment in BluefruitLE_SPI_Stream.cpp to clarify why it, too, is empty.